app/frontend/stylesheets/autocomplete/_autocomplete.scss

```diff
@@ -0,0 +1,82 @@
+// Overrides for the autocomplete js plugin.
```

Comment:
Was it generated in a way? I can see it's not using the SASS features

Reply:
I didn't generate it but I copied the selectors from the chrome dev tools as it was a pain. I'll tidy them up :)